static/styles/twodsix_basic.css

```diff
@@ -516,9 +516,18 @@ a.skill-tab.active, a.item-tab.active, a.finances-tab.active, a.info-tab.active,
 
 .items-traits {
   display: grid;
-  grid-template-columns: 14em 2em 14em 3em;
+  grid-template-columns: 7em 22em 4em;
```

Comment:
how about 
"grid-template-columns: 8em 19em 4em;"
so things don't run to the edge